recipe/meta.yaml

```diff
@@ -1,37 +1,61 @@
-{% set version = "0.7.0" %}
-{% set sha256 = "1aaadcd69338252ade4f7ec1265e1a19184bf916d84c9b7df095f423948cb89f" %}
+{% set version = "0.10.0" %}
 
 package:
   name: build
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/b/build/build-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_url: https://github.com/pypa/build
+  git_rev: {{ version }}
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - pyproject-build = build.__main__:entrypoint
 
 requirements:
+  build:
+    - git  # [not win]
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
+    - flit-core 3.8.0
   run:
-    - python >=3.6
-    - packaging
-    - pep517 >=0.9.1
-    - tomli
-    - importlib-metadata
+    - python
+    - packaging >=19.0
+    - pyproject_hooks
+    - importlib-metadata >=0.22  # [py<38]
+    - tomli >=1.1.0  # [py<311]
+
+{% set tests_to_skip = "test_build_wheel or test_build_sdist" %}
+{% set tests_to_skip = tests_to_skip + " or test_init" %}  # [linux]
+{% set tests_to_skip = tests_to_skip + " or test_output_env_subprocess_error" %}  # [not win]
 
 test:
+  source_files:
+    - src/build/
```

Comment:
Why is `src/build` needed? These files should already be installed in the package...

Reply:
I remember trying to run the tests without these files and they wouldn't run.
Furthermore, [this](https://github.com/pypa/build/blob/cd06da25481b9a610f846fa60cb67b5a5fa9a051/pyproject.toml#L80C1-L80C1).
I will try again without the build dir and see if they indeed don't run.

Reply:
https://github.com/pypa/build/blob/cd06da25481b9a610f846fa60cb67b5a5fa9a051/pyproject.toml#L80 is only for code coverage purposes. We don't run tests with coverage enabled.

Reply:
let me try it one time without it and then I'll remove it.